### ISSUE TITLE
Replace deis references in annotations

### ISSUE THREAD

**Issue by jdolitsky** [oras-project/oras-go#4]
See https://github.com/oras-project/oras-go/search?q=deis

cc @shizhMSFT

**Comment by shizhMSFT** [comment]
Any suggestions on the new annotation keys? /cc @SteveLasker

**Comment by SteveLasker** [comment]
How about `cncf.oras.*`
What are these used for?

**Comment by shizhMSFT** [comment]
They are used for compressed directories. If we adopt #178, then we don't need to bother those annotations.

**Comment by SteveLasker** [comment]
Of course, I'd like to see #178, but I wouldn't block and seems you can use `cncf.oras`, however that does presume the project is accepted, so I'd suggest holding this annotation for now. It also makes it interesting how these would work if a new client is looking for `cncf.oras` and it discovers `deis.oras`?
Are you thinking of maintaining some backwards compat?

**Comment by bacongobbler** [comment]
Is there a risk to users if these annotations are changed and/or removed? As in, will Helm and other applications using oras be able to unpack charts using these annotations?

I'm okay with changing the annotation - just need a heads up so we can inform users of breaking changes to `helm chart push/helm chart pull`.

**Comment by jdolitsky** [comment]
@bacongobbler Helm's use of oras-go (both the old version and the one in the HIP 6 PR) does not do anything with these annotations, so we are safe here

**Comment by FeynmanZhou** [comment]
I think we can add the website domain name as an option in annotations, then we have two options:
- `cncf.oras`
- `land.oras` 

How about we vote one and then provide a backward compact solution? @SteveLasker @bacongobbler @shizhMSFT @sajayantony @yizha1
